internal/stack/_static/logstash.conf.tmpl

```diff
@@ -4,7 +4,7 @@ input {
     ssl_enabled => true
     ssl_certificate_authorities => ["/usr/share/logstash/config/certs/ca-cert.pem"]
     ssl_certificate => "/usr/share/logstash/config/certs/cert.pem"
-    ssl_key => "/tmp/logstash.pkcs8.key"
+    ssl_key => "/usr/share/logstash/config/certs/key.pem"
```

Comment:
@bhapas was there a reason to convert to PKCS8? It seems to work with the current key.

If for some reason we need to convert, I can revert this change https://github.com/elastic/elastic-package/pull/1687/commits/b1f174b9d05d5fb4ea41af6fe454115358dd89d1 where I was doing it when generating the certificates.

Reply:
+1,
I have just checked that  `key.pem` is `PKCS#8` formatted.

Let's also change the `serverless-logstash.conf.tmpl` file as well.